nkms_eth/actors.py

```diff
@@ -0,0 +1,263 @@
+from collections import OrderedDict
+from typing import Tuple, List
+
+# from nkms_eth.agents import MinerAgent, PolicyAgent
+from nkms_eth.base import Actor
+
+
+class PolicyArrangement:
+    def __init__(self, author: 'PolicyAuthor', miner: 'Miner', value: int,
+                 periods: int, arrangement_id: bytes=None):
+
+        if arrangement_id is None:
+            self.id = self.__class__._generate_arrangement_id()  # TODO: Generate policy ID
+
+        # The relationship exists between two addresses
+        self.author = author
+        self.policy_agent = author.policy_agent
```

Comment:
Why do we need to directly compose the `policy_agent` (as opposed to always using `self.author.policy_agent`)?

Reply:
Just because it follows the same convention as other entities. I think I agree that it's more semantically rich as `self.author.policy_agent`.